test/cli/html-reporter.test.js

```diff
@@ -0,0 +1,49 @@
+var fs = require('fs');
+
+describe('HTML reporter', function () {
```

Comment:
These tests don't actually test the generated files, so I think we should name the file accordingly. `shallow-html-reporter.test.js` maybe? @kunagpal